apps/masterbots.ai/lib/helpers/ai-helpers.ts

```
@@ -27,7 +28,7 @@ export function getModelClientType(model: AIModels) {
 export function createPayload(
   json: { id: string },
   messages: { content: string }[],
-  completion: any
+  completion: any,
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)